cfme/tests/candu/test_utilization_metrics.py

```diff
@@ -438,3 +461,34 @@ def test_utilization_breadcrumbs(appliance):
     view = navigate_to(appliance.collections.utilization, "All")
     assert view.breadcrumb.locations == BREADCRUMB_LOCATIONS["OverviewUtilization"]
     assert view.breadcrumb.is_displayed
+
+
+@pytest.mark.tier(1)
+@pytest.mark.provider([VMwareProvider], scope="module")
+@pytest.mark.parametrize("candu_db_restore", ["utilization_db.backup"], ids=["db"], indirect=True)
+def test_utilization_utilization_graphs(temp_appliance_extended_db, candu_db_restore, request,
+        entity):
+    """
+    Polarion:
+        assignee: gtalreja
+        casecomponent: Optimize
+        initialEstimate: 1/4h
+        testSteps:
+            1. Enable C&U
+            2. Wait until data will be collected
+            3. Go to Optimize/Utilization
+        expectedResults:
+            1.
+            2.
+            3. Verify that all graphs shows correctly
+    """
+    view = navigate_to(entity, "UtilTrendSummary")
+
```

Comment:
Since `entity` is set up on the temporary appliance, the test should run `navigate_to` and the `view` methods within a `with temp_appliance_extended_db` context block so that the browser actually points to the temp appliance.

Reply:
Yes, it will make more sense, and this will make execution faster, as for every entity it was refreshing a browser for pointing to a temp_appliance again, thanks @tpapaioa :)